Header header
string[] axis_names
PoseTrajectoryPoint[] points
